### ISSUE TITLE
python package netapp-lib dependencies are broken

### ISSUE THREAD

**Issue by phospi** [ansible-collections/netapp.ontap#12]
##### SUMMARY
So, this is not a bug of the collection but of the python on which this collection relies. But I was not able to identify the proper way to raise issues for the python package. The pypi page is a huge void.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
netapp-lib==2020.7.16

##### PYTHON VERSION
Tested with python 3.6.8

##### COLLECTION VERSION
all versions with the following code in `plugins/module_utils/netapp.py`
````
try:
    from netapp_lib.api.zapi import zapi
    HAS_NETAPP_LIB = True
except ImportError as exc:
    HAS_NETAPP_LIB = False
    IMPORT_EXCEPTION = exc
````

##### STEPS TO REPRODUCE
Create a pipenv of your choice. Install `netapp-lib`. Run the import `from netapp_lib.api.zapi import zapi`

Example:
````
mkdir /tmp/py-netapp
cd /tmp/py-netapp
pipenv install
pipenv shell
(py-netapp) pip install netapp-lib
(py-netapp) python -c "from netapp_lib.api.zapi import zapi"
````

##### EXPECTED RESULTS
Empty return. Indicating a successful import.

##### ACTUAL RESULTS

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/root/.local/share/virtualenvs/netapp_env-dPj_okWD/lib/python3.6/site-packages/netapp_lib/api/zapi/zapi.py", line 17, in <module>
    import six
ModuleNotFoundError: No module named 'six'
```

##### MY OPINION
Manually installing `six` solves the problem. netapp-lib dependencies must be fixed to automatically install `six`. Else the collection will return `Error: the python NetApp-Lib module is required. <python error>`.

**Comment by lonico** [comment]
Yes, we improved the error reporting in the collection, so that it shows `ModuleNotFoundError: No module named 'six'`.

The real fix will be to update netapp-lib with an explicit dependency on six.   It was missed as six was present by default on earlier versions.

**Comment by lonico** [comment]
work-around:
```
pip install six
```

**Comment by lonico** [comment]
fixed in netapp-lib 2021.6.25  (available in pypi)
